### PR TITLE
OBPIH-4700 Fetch order item summary by order instead of for each item

### PR DESCRIPTION
**Pull request by awalkowiak**

### CODE REVIEW

File: grails-app/views/order/_orderSummary.gsp [hunk 1]
```diff
@@ -76,10 +76,10 @@
...
             <g:each var="orderItem" in="${orderInstance?.orderItems?.sort { a,b -> a.dateCreated <=> b.dateCreated ?: a.orderIndex <=> b.orderIndex }}" status="i">
-                <g:set var="isItemCanceled" value="${orderItem.orderItemStatusCode == OrderItemStatusCode.CANCELED}"/>
-                <g:if test="${!isItemCanceled || orderInstance?.orderType==OrderType.findByCode(OrderTypeCode.PURCHASE_ORDER.name())}">
-                    <tr class="order-item ${(i % 2) == 0 ? 'even' : 'odd'} dataRow" style="${isItemCanceled ? 'background-color: #ffcccb;' : ''}">
+                <g:if test="${!orderItem?.canceled || orderInstance?.isPurchaseOrder}">
+                    <tr class="order-item ${(i % 2) == 0 ? 'even' : 'odd'} dataRow" style="${orderItem?.canceled ? 'background-color: #ffcccb;' : ''}">
                         <td>
                             <div class="tag ${orderItem?.canceled ? 'tag-danger' : ''}">
                                 <format:metadata obj="${orderItem?.canceled ? orderItem?.orderItemStatusCode?.name() : orderItem?.getOrderItemStatus()}"/>
```
Comment (jmiranda) [inline grails-app/views/order/_orderSummary.gsp:85]: There's a call to getOrderItemStatus() here as well.

Comment (awalkowiak) [reply grails-app/views/order/_orderSummary.gsp:85]: oof I missed that one

Comment (awalkowiak) [reply grails-app/views/order/_orderSummary.gsp:85]: wait, i pushed "non-fixed" version :see_no_evil:

Comment (awalkowiak) [reply grails-app/views/order/_orderSummary.gsp]: @jmiranda Fixed

**Review by jmiranda — APPROVED** [review]
This looks good.

**Pull request merged**